python/pyarrow/compute.py

```diff
@@ -590,6 +590,38 @@ def take(data, indices, *, boundscheck=True, memory_pool=None):
     return call_function('take', [data, indices], options, memory_pool)
 
 
+def dropnull(data, *, memory_pool=None):
```

Comment:
Is this explicit function needed? (the auto-generated function doesn't work, or are the vector functions not auto-generated?)

Reply:
To the best of my knowledge, this function is needed to be exposed as part of `pyarrow.compute` module https://github.com/apache/arrow/blob/082b11e9ecb4acd60d2e36ba8b5b3bda8b2027ce/python/pyarrow/lib.pyx#L113

Reply:
I quickly tried it out, and also without this explicit function, the auto-generated one seems to work fine. 

cc @pitrou do you know on what basis it's decided when to use the auto-generated one and when to add an explicit function?

Reply:
The explicit functions are mostly historical (auto-generation became less and less limited over time), but in some cases they allow for more user-friendly signatures. If your explicit function has the same signature as the auto-generated one, then it's basically useless :-)

Reply:
I think the explicit function can thus be removed.